client/src/components/Meeting/Meeting.tsx

```diff
@@ -1,66 +1,112 @@
-        <Typography variant="h5">{loggedInUser.email}</Typography>
-        <Button variant="outlined" color="primary" href="#outlined-buttons" className={classes.right}>
-          <Typography variant="h6">
+    <Box>
+      <Grid container xs={12} direction="row" className={classes.profileSection}>
+        <Box display="flex">
+          <Box>
+            <AvatarDisplay loggedIn user={loggedInUser} />
+          </Box>
+          <Box marginLeft={2}>
+            <Typography component="h3" className={classes.title}>
+              {loggedInUser.username}
+            </Typography>
+            <Typography variant="subtitle1">{loggedInUser.email}</Typography>
+          </Box>
+        </Box>
+        <Box>
+          <Button variant="outlined" color="primary" className={classes.newEventButton}>
             <AddIcon />
-            New Event
-          </Typography>
-        </Button>
+            <Typography variant="h6">New Event</Typography>
+          </Button>
+        </Box>
       </Grid>
-      <Grid item xs={12}>
-        <Grid container direction="row" alignItems="center" justify="center" spacing={spacing}>
-          {[0, 1, 2].map((value) => (
-            <Grid key={value} item xs={3}>
-              <Paper className={classes.paper}>
-                <Typography variant="h5">15 min Meeting</Typography>
-                <SettingsIcon />
-                <Divider />
-                <ScheduleIcon />
-                <Typography variant="h5">15</Typography>
-                <Button variant="outlined" color="primary">
-                  <Link to={`/${loggedInUser.username}/15min`}>Create Meeting</Link>
-                </Button>
-              </Paper>
-            </Grid>
-          ))}
-        </Grid>
-        <Button variant="outlined" color="primary" href="#outlined-buttons" className={classes.right}>
-          Getting Started!
-        </Button>
+      <Divider className={classes.divider} />
+      <Grid container xs={12} className={classes.meetingContainer} spacing={3}>
+        {events?.map((eachEvent: any, index: any) => (
+          <Grid key={index} item xs={12} md={4}>
+            <Box className={classes.cardStylingBar} style={{ backgroundColor: circularColorPicker(index) }}></Box>
+            <Paper className={classes.meetingCard}>
+              <Box marginBottom={2} textAlign="left">
+                <Box display="flex" justifyContent="flex-end">
+                  <SettingsIcon />
+                </Box>
+                <Box paddingX={2}>
+                  <Typography variant="h5" className={classes.title}>
+                    {eachEvent.name}
+                  </Typography>
+                  <Typography variant="body1">{eachEvent.description}</Typography>
+                </Box>
+              </Box>
+              <Box>
+                <Divider className={classes.divider} />
+                <Box display="flex" justifyContent="space-between">
```

Comment:
Could utilize Grid container here and Grid item. When working with Material UI when we want to do Flexbox work it's common to do so with Grid